MAINTAINERS.yml

```diff
@@ -0,0 +1,1052 @@
+
+"Drivers: I2C":
+    # Provisional entry, to be corrected
+    # TODO: spreadsheet not matching CODEOWNERS
+    status: maintained
+    maintainers:
+        - mbolivar
+    files:
+        - drivers/i2c/
+        - dts/bindings/i2c/
+        - include/i2c.h
+        - include/drivers/i2c.h
+        - include/drivers/i2c/
+    labels:
+        - "area: I2C"
+
+"Drivers: I2S":
+    # Provisional entry, to be corrected
+    status: orphaned
+    files:
+        - doc/reference/peripherals/i2s.rst
+        - drivers/i2s/
+        - dts/bindings/i2s/
+        - include/i2s.h
+        - include/drivers/i2s.h
+        - tests/drivers/i2s/
+    labels:
+        - "area: I2S"
+
+"Drivers: LED":
+    # Provisional entry, to be corrected
+    status: orphaned
+    collaborators:
+        - Mani-Sadhasivam
+    files:
+        - drivers/led/
+        - include/led.h
+        - include/drivers/led/
+        - include/drivers/led.h
+        - samples/drivers/led_*/
+    labels:
+        - "area: LED"
+
+"Drivers: LED Strip":
+    # Provisional entry, to be corrected
+    status: orphaned
+    collaborators:
+        - mbolivar
+    files:
+        - drivers/led_strip/
+        - dts/bindings/led_strip/
+        - include/led_strip.h
+        - include/drivers/led_strip.h
+
+"Drivers: Modem":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - mike-scott
+    files:
+        - drivers/modem/
+    labels:
+        - "area: Modem"
+
+"Drivers: Neural networking":
+    # Provisional entry, to be corrected
+    status: orphaned
+    files:
+        - drivers/neural_net/
+
+"Drivers: PCI":
+    # Provisional entry, to be corrected
+    # TODO: spreadsheet not matching CODEOWNERS
+    status: maintained
+    maintainers:
+        - dcpleung
+    collaborators:
+        - andrewboie
+    files:
+        - drivers/pcie/
+        - include/drivers/pcie/
+    labels:
+        - "area: PCI"
+
+"Drivers: Pinmux":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - mnkp
+    files:
+        - doc/reference/peripherals/pinmux.rst
+        - drivers/pinmux/
+        - include/drivers/pinmux.h
+        - include/pinmux.h
+        - tests/drivers/pinmux/
+    labels:
+        - "area: Pinmux"
+
+"Drivers: PWM":
+    # Provisional entry, to be corrected
+    status: orphaned
+    files:
+        - drivers/pwm/
+        - dts/bindings/pwm/
+        - include/drivers/pwm.h
+        - include/pwm.h
+        - tests/drivers/pwm/
+    labels:
+        - "area: PWM"
+
+"Drivers: Serial/UART":
+    # Provisional entry, to be corrected
+    # TODO: spreadsheet not matching CODEOWNERS
+    status: maintained
+    maintainers:
+        - dcpleung
+    files:
+        - drivers/serial/
+    labels:
+        - "area: UART"
+
+"Drivers: Sensors":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - MaureenHelm
+    files:
+        - drivers/sensor/
```

Comment:
I maintain all the ST sensors:

/drivers/sensor/hts*/  
/drivers/sensor/lis*/ 
/drivers/sensor/lps*/  
/drivers/sensor/lsm*/  
/drivers/sensor/st*/ 

Not sure how to handle this.
Maybe you can put me as collaborator if Maureen agrees.

Reply:
Thanks @avisconti . The best thing is of course to agree with @MaureenHelm. Maybe you want to have a new area entry with those(?). Then you could either be set as maintainer of that sub-area, or just a collaborator. You just tell me the changes you want, and I add them here as a separate commit.

Reply:
@aescolar 
I think it should be difficult to separate the ST part from the rest (not sure '*' is allowed).
Probably Collaborator of the 'sensor' part (driver, samples, ...) would be fine.
Let's see what @MaureenHelm has to say.

Reply:
> (not sure '*' is allowed)

It certainly is :). You may want to look into one of the entries or more complex subsystems today, like BT, for some examples of what is possible

Reply:
@aescolar 
Verified with @MaureenHelm .
You can put me as Collaborator for sensors